libraries/c_sdk/standard/mqtt/include/mqtt_config.h

```diff
@@ -22,11 +22,18 @@
 
 /************ End of logging configuration ****************/
 
+/* Set network context used for sending the packets on the network. */
+typedef struct NetworkContext
```

Comment:
Do we need to do a typedef here? I believe it was already defined as a type in transport_interface.h